pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/Project.java

```diff
@@ -773,6 +774,36 @@ private PropertyLocation getPropertyDefinition( Log log, String propertyName, bo
 		return null;
 	}
 
+	/**
+	 * @return All submodules of this project or an empty list if it does not have any.
+	 */
+	public Stream<Project> getSubmodules()
```

Comment:
Everywhere else we use List<>, let's keep on with that.
Better to have the same kind of API everywhere

Reply:
Fair enough, I am going to change that